src/pages/Wishlist/Wishlist.jsx

```diff
@@ -1,6 +1,6 @@
 import { ProductCard } from "../../components";
 import { useWishlist } from "../../context";
```

Comment:
Suggestion: You can use absolute paths in imports.
It will look cleaner. Surbhi suggested me in my PR.

https://stackoverflow.com/questions/41993461/import-react-components-with-absolute-path

Reply:
yeah i had used it earlier but you could also use barrel export as using import can become verbose.
you can have a read here.
https://dev.to/justmyrealname/organize-react-components-better-with-barrel-exports-4fno

thanks